stdlib/bytes.gr

```diff
@@ -367,6 +367,24 @@ export let rec fill = (value: Int32, bytes: Bytes) => {
   ret
 }
 
+/**
+ * Replaces all bytes in a byte sequnce with value zero.
+ *
+ * @param bytes: The byte sequence to clear
+ *
+ * @since v0.3.2
+ */
+@disableGC
+export let rec clear = (bytes: Bytes) => {
+  let (+) = WasmI32.add
+  let src = WasmI32.fromGrain(bytes)
+  let size = getSize(src)
+  let ret = Memory.fill(src + _VALUE_OFFSET, 0n, size)
+  Memory.decRef(WasmI32.fromGrain(bytes))
+  Memory.decRef(WasmI32.fromGrain(clear))
+  ret
```

Comment:
Since `fill` returns void, this feels a little weird. I think this is a bit more clear:
```suggestion
  Memory.fill(src + _VALUE_OFFSET, 0n, size)
  Memory.decRef(WasmI32.fromGrain(bytes))
  Memory.decRef(WasmI32.fromGrain(clear))
  void
```

Reply:
Oops. I blindly copy and pasted from the `fill` function which does the same thing, returning void.